src/components/core/DpDataTable/DpTableHeader.vue

```diff
@@ -2,56 +2,61 @@
   <tr
     ref="tableHeader"
     :class="{ 'c-data-table__sticky-header': isSticky }">
+    <th
+      v-if="isDraggable"
+      scope="col"
+      class="c-data-table__cell--narrow">
+      <dp-icon
+        class="c-data-table__drag-handle"
+        icon="drag-handle" />
+    </th>
+    <th
+      v-if="isSelectable"
+      scope="col"
+      class="c-data-table__cell--narrow">
+      <input
+        :aria-label="this.translations.headerSelectHint"
+        :title="this.translations.headerSelectHint"
+        type="checkbox"
+        data-cy="selectAll"
+        ref="selectAll"
+        @click="$listeners.toggleSelectAll()"
+        :checked="checked" />
+    </th>
     <template v-for="(hf, idx) in headerFields">
       <dp-resizable-column
         v-if="isResizable"
         :is-last="headerFields.length === idx"
         :header-field="hf"
         :idx="idx">
-        {{ hf.label }}
+        <slot :name="`header-${hf.field}`">
```

Comment:
instead of this way, I would suggest to write it this way:
`<slot :header-field="hf.field"`>
 and change it in the files, too where it has the reference
 because the problem is, this way the code is not so readable after a few time

Reply:
you mean not readable in Vue 3? Would it be something like `<slot :[`header-${field}`]="hf.field">`? Because `<slot :header-field="hf.field">` dose not work

Reply:
:[\`header-${field}\`]="hf.field"  sounds good! (if it works)

Reply:
I see now. lgtm thanks for the affort